Tests/scripts/install_content_and_test_integrations.sh

```diff
@@ -1,9 +1,23 @@
 #!/usr/bin/env bash
 
+function exit_on_error {
```

Comment:
Can't it be in one place?

Reply:
nope, it's too complex. i will refactor it when we move the scripts out of content